### PR TITLE
{bio}[foss/2023a] scCODA v0.1.9, rpy2 v3.5.15, scikit-bio v0.5.9, ...

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @pavelToman!

File: easybuild/easyconfigs/s/scikit-bio/scikit-bio-0.5.9-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonBundle'
+
+name = 'scikit-bio'
+version = '0.5.9'
+local_commit = '7565847'
```
Comment (boegel) [inline easybuild/easyconfigs/s/scikit-bio/scikit-bio-0.5.9-foss-2023a.eb:5]: This commit doesn't correspond to the `0.5.9` release at all, I've opened https://github.com/easybuilders/easybuild-easyconfigs/pull/20300 to rectify that...

**Pull request merged**